src/layout/header/LanguageToggle.tsx

```diff
@@ -12,12 +12,12 @@ export const LanguageToggle = () => {
 
   return (
     <button
-      className="theme-icon"
+      className="header-link"
       onClick={handleChangeLanguage}
       aria-label={t('Change Language')}
       title={t('Change Language')}
-      style={{ margin: '1em', border: 'none', background: 'transparent', cursor: 'pointer' }}>
-      <GlobalOutlined style={{ fontSize: '1.7em' }} />
+      style={{ border: 'none', background: 'transparent', cursor: 'pointer' }}>
```

Comment:
it could be nice to move this inline css code to a css file

Reply:
Updated, thanks!